esx_service/utils/auth_data.py

```diff
@@ -611,22 +676,18 @@ def get_tenant(self, tenant_name):
                 # search privileges and default_privileges for this tenant
                 privileges = []
                 cur = self.conn.execute(
-                    "SELECT * FROM privileges WHERE tenant_id = ? AND datastore != ?",
-                    (id,default_datastore)    
+                    "SELECT * FROM privileges WHERE tenant_id = ?",
+                    (id,)    
                 )
                 privileges = cur.fetchall()
 
                 logging.debug("privileges=%s", privileges)
-
-                default_privileges = []
-                cur = self.conn.execute(
-                    "SELECT * FROM privileges WHERE tenant_id = ? AND datastore = ?",
-                    (id,default_datastore)    
-                )
-                default_privileges = cur.fetchall()
-                logging.debug("default_privileges=%s", default_privileges)
-                tenant = DockerVolumeTenant(name, description, default_datastore,
-                                            default_privileges, vms, privileges, id)
+                tenant = DockerVolumeTenant(name=name,
+                                            description=description,
+                                            vms=vms,
+                                            privileges=privileges,
```

Comment:
The current privileges field contains specific DB column information. As we discussed offline, the API layer should return a pure data object so that above layer (such as VMODL) can get the data directly without having dependency on DB specifics.